### ISSUE TITLE
Email configuration not taken into consideration and no mail menu in the AppURL/dashboard/settings/mail

### ISSUE THREAD

**Issue by Djas2017** [CachetHQ/Cachet#2594]
Hi,

I followed the documentation to install Cachet on CentOS 7, all the tables in the DB seem to be created.

- The Subscribe button is visible and people can add their email address.
- Subscribed people do no receive an email to confirm their email address.
- The Email configuration not taken into consideration (from the .env file)
-  and no mail menu in the AppURL/dashboard/settings/mail 

Snippet from the .env file:
...
CACHE_DRIVER=apc
SESSION_DRIVER=apc
QUEUE_DRIVER=database

MAIL_DRIVER=smtp
MAIL_HOST=smtp.zoho.com
MAIL_PORT=587
MAIL_USERNAME=noreply@mydomain.com
MAIL_PASSWORD=Pa5sW0rdHere
MAIL_ADDRESS=noreply@mydomain.com
MAIL_NAME=Status
MAIL_ENCRYPTION=tls
...

**Comment by jbrooksuk** [comment]
You need to run the [queue](https://docs.cachethq.io/docs/configuring-the-queue).

**Comment by Djas2017** [comment]
Tested on multiple servers (ubuntu, centos) and finally works!!!
 
Email notification and subscription works now fine without configuring anything as described in https://docs.cachethq.io/docs/configuring-the-queue. The only thing I did was to follow the installation instructions and configured https://docs.cachethq.io/docs/configuring-mail in the .env file. Did you add anything to v2.4 which processes emails automatically or ...? 
For ref I have installed a new instance cachet 2.4 on CentOS 7.3

PS: Had issues when running php artisan migrate but changing the public function boot() in AppServiceProvider.php before running  php artisan migrate and reverting the changes after that fixed the issue.  See https://github.com/CachetHQ/Cachet/issues/2588
